docs/basics/faq-troubleshooting.md

```diff
@@ -87,29 +92,40 @@ for [`torrentDir`](./options.md#torrentdir) temporarily. This will strictly sear
 
 #### v6
 
-If possible, `cross-seed` will always attempt to create links for torrents being injected into a client, unlike in v5.
-This includes torrent-based searches as well as data-based.
+If possible, `cross-seed` will always attempt to create links for data and torrents being injected into a client, unlike in v5 where torrent matches were directly linked to the data.
+
+:::tip
+If you wish to search only data, please read the relevant version's instructions.
+
+-   [v5](#v5)
+-   [v6](../v6-migration.md#updated-torrentdir-option)
+
+:::
 
 ### Error: ENOENT: no such file or directory
 
 This is usually caused by broken symlinks in, or files being moved or deleted from, your [`dataDirs`](./options.md#datadirs) during a data-based search.
 Check the `/logs/verbose.*.log` files for the file causing this.
 
-:::tip
+:::caution
 If you do not link files within your `dataDirs` or have them outside of the [`maxDataDepth`](../tutorials/data-based-matching.md#setup) visibility, this is preventable.
 :::
 
 ### My torrents are injected (qBittorrent) but show `missing files` error!
 
 If you are seeing injected torrents show up with `missing files` error, it is likely because you do not have a save path set for the original torrent's category that `cross-seed` was cross-seeding from. You should also see a warning when the torrents are being injected telling you that there is no save path set.
```

Comment:
the difference between tip and caution is only the icon for the box, and the color. You can add whatever word you want in place of Tip by just doing the following..

`:::tip Caution`

is there a reason you want this color changed? It's a tip imo

Reply:
I'm not asking for a color change. I'm asking about the paragraph of text where is says:
> You should also see a warning when the torrents are being injected telling you that there is no save path set

Reply:
Oh sorry, I just woke up...I thought that was the suggestion lol

It'd be in the logs that the warning would exist for cross-seed, or where the progress bar is in qbittorrent...

Reply:
```suggestion
If you see injected torrents show up with a `missing files` error, it is likely because you do not have a save path set for the original torrent's category that `cross-seed` was cross-seeding from. You should also see a warning in the logs when the torrents are injected telling you there is no save path set.
```